### PR TITLE
GCS Data Lake documentation update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated connection settings for GCS data lake

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [GCS datalake doc updates for new fields](https://www.notion.so/rudderstacks/Gcs-Datalake-documentation-for-new-fields-8b225dbe2900428eb52d89692b03a36e)

### CODE REVIEW

File: docs/data-warehouse-integrations/gcs-datalake.mdx [hunk 1]
```diff
@@ -73,15 +73,30 @@ To configure GCS data lake as a destination in RudderStack, follow these steps:
...
-    - **Sync Frequency**: Specify how often RudderStack should sync the data to your GCS data lake.
-    - **Sync Starting At**: This optional setting lets you specify the particular time of the day (in UTC) when you want RudderStack to sync the data to the data lake.
+
+- **GCS Storage Bucket Name**: The name of the GCS bucket used to store data before loading it into the GCS data lake.
+- **Prefix**: If specified, RudderStack will create a folder in the bucket with this prefix and push all the data within that folder. For example, `https://storage.googleapis.com/<bucketName>/<prefix>/`.
+- **Namespace**: If specified, all the data for the destination will be pushed to `https://storage.googleapis.com/<bucketName>/<prefix>/rudder-datalake/<namespace>`. If you don't specify a namespace in the settings, RudderStack sets it to the source name, by default.
+- **Table Suffix**: This optional setting lets you define a custom path under which your table data is stored. For example, if you set this field to `rs`, your data will be pushed to `https://storage.googleapis.com/<bucketName>/<prefix>/rudder-datalake/<namespace>/<tableName>/rs`.
```
Comment (ishachopra0401) [inline docs/data-warehouse-integrations/gcs-datalake.mdx:84]:
```suggestion
- **Table Suffix**: This optional setting lets you define a custom path under which your table data is stored. For example, if you set this field to `rs`, your data will be pushed to `https://storage.googleapis.com/<bucketName>/<prefix>/rudder-datalake/<namespace>/<table-name>/rs`.
```

Should this be `table-name` as in the UI and also in the location mentioned in the below setting?

Comment (ameypv-rudder) [reply docs/data-warehouse-integrations/gcs-datalake.mdx:84]: @Jayachand can you please confirm this? In the example you shared, snake case was mentioned.

File: docs/data-warehouse-integrations/gcs-datalake.mdx [hunk 2]
```diff
@@ -73,15 +73,30 @@ To configure GCS data lake as a destination in RudderStack, follow these steps:
...
+- **Choose time window layout**: This option lets you set the timestamp-defined layout structure under which the table data will be stored.
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/data-warehouse-integrations/gcs-datalake.mdx:85]: makes sense. I will add the screenshot.

File: docs/data-warehouse-integrations/gcs-datalake.mdx (hunk 1 above)
Comment (Jayachand) [reply docs/data-warehouse-integrations/gcs-datalake.mdx:84]: lets have it consistent with UI

**Review by Jayachand — APPROVED** [review]
LGTM

**Pull request merged**